### PR TITLE
Dont clear ongoing activities

### PR DESCRIPTION
**Pull request by 0x6431346e**
## What changed?
Insert here the changes.

### CODE REVIEW

**Comment by koalalorenzo** [comment]
Should we add some tests about this?

File: app/index.js [hunk 1]
```diff
@@ -362,9 +362,31 @@ ipcMain.on('update-activities', () => {
   updateActivitiesWindow()
 })
 
+export const handleClearActivities = (activitiesById, activities) => {
```
Comment (koalalorenzo) [inline app/index.js]: can we add more comments?

Comment (koalalorenzo) [reply app/index.js]: We need a little more documentations on what this is doing for future works

Comment (0x6431346e) [reply app/index.js]: Yep, good idea :taco:

Comment (0x6431346e) [reply app/index.js]: Does this make more sense?

Comment (koalalorenzo) [reply app/index.js]: Yes! :D

**Pull request merged**